client/src/partials/reports_proposed/variation_exploitation/variation_exploitation.js

```diff
@@ -0,0 +1,69 @@
+var variationExploitationController = function ($http, $translate, validate, reportConfigService, messenger) {
+  var dependencies = {}, generatedDocumentPath = null, serverUtilityPath = 'report/build/variation_exploitation', 
+    configuration = reportConfigService.configuration, vm = this;
+
+  dependencies.fiscalYears = {
+    query : {
+      identifier : 'id',
+      tables : {
+        'fiscal_year' : {
+          columns : ['id', 'fiscal_year_txt', 'previous_fiscal_year']
+        }
+      }
+    }
+  };
+
+  vm.generate_doc = $translate.instant('VARIATION_EXPLOITATION.GENERATE_DOC');
+  vm.loading = $translate.instant('VARIATION_EXPLOITATION.LOADING');
+  vm.building = false;
+  vm.configuration = configuration;
```

Comment:
It's a bit confusing to bind a series of properties to `vm` at both the top and [bottom](https://github.com/IMA-WorldHealth/bhima/pull/837/files#diff-516a7d58993316077d75d5d7e37c182fR63) of your controller function.  Could you choose one location for the bindings?